zipline/algorithm.py

```diff
@@ -885,19 +1023,51 @@ def symbol(self, symbol_str):
 
     @api_method
     def symbols(self, *args):
-        """
-        Default symbols lookup for any source that directly maps the
-        symbol to the Asset (e.g. yahoo finance).
+        """Lookup multuple Equities as a list.
+
+        Parameters
+        ----------
+        *args : iterable[str]
+            The ticker symbols to lookup.
+
+        Returns
+        -------
+        equities : list[Equity]
+            The equities that held the given ticker symbols on the current
+            symbol lookup date.
+
+        Raises
+        ------
+        SymbolNotFound
+            Raised when one of the symbols was not held on the current
+            lookup date.
+
+        See Also
+        --------
+        :func:`zipline.api.set_symbol_lookup_date`
         """
         return [self.symbol(identifier) for identifier in args]
 
     @api_method
-    def sid(self, a_sid):
-        """
-        Default sid lookup for any source that directly maps the integer sid
-        to the Asset.
+    def sid(self, sid):
+        """Lookup an Asset by its unique asset identifier.
+
+        Parameters
+        ----------
+        sid : int
+            The unique integer that identifies an asset.
+
+        Returns
+        -------
+        asset : Asset
+            The asset with the given ``sid``.
+
+        Raises
+        ------
+        SidsNotFound
+            When a requested sid is not found and default_none=False.
```

Comment:
not sure. this was already here but I will remove it. I think that comment is taken from the asset finder.